### DISCUSSION TITLE
Aptos Connect in Telegram returned client exception error.

### DISCUSSION THREAD

**Discussion by ogdev97** [aptos-labs/aptos-developer-discussions#482] [Dev Doc and Tutorial Feedback]
Trying to connect with aptos connect for my self developed telegram web-app , but it returns this client exception error. Any idea how to fix it? More info : I try with mizu wallet and it worked.

![image](https://github.com/user-attachments/assets/ee0c773f-868e-4314-90c5-dbb5e4ca8c8e)

**Comment by 0xmaayan** [comment]
Hi @angangang97, thanks for your report! Could you clarify what do you use to connect AptosConnect/Mizu in your TG app? Do you use the Aptos Wallet Adapter? If so, what version? Also, Do you use the create-aptos-dapp TG template?

Thanks!

**Comment by ogdev97** [comment]
Hey @0xmaayan sure, this is my wallet-adapter version. I use the normal aptos wallet adapter in this page. https://aptos.dev/en/build/sdks/wallet-adapter/dapp

`  "dependencies": {
    "@aptos-labs/ts-sdk": "^1.22.1",
    "@aptos-labs/wallet-adapter-react": "^3.7.0",`
    
 Below is my WalletProvider.tsx code if u need references : 
 
'use client';

import { AptosWalletAdapterProvider } from '@aptos-labs/wallet-adapter-react';

import { PropsWithChildren } from 'react';
import { Network } from '@aptos-labs/ts-sdk';

import { useToast } from './ui/use-toast';

export const WalletProvider = ({ children }: PropsWithChildren) => {
  const { toast } = useToast();

  return (
    <AptosWalletAdapterProvider
      dappConfig={{
        network: Network.TESTNET,
        aptosConnectDappId: '',
        mizuwallet: {
          manifestURL:
            'https://assets.mz.xyz/static/config/mizuwallet-connect-manifest.json'
        }
      }}
      onError={(error) => {
        toast({
          variant: 'destructive',
          title: 'Error',
          description: error || 'Unknown wallet error'
        });
      }}
    >
      {children}
    </AptosWalletAdapterProvider>
  );
};
